identity/gcp-gce-auth/README.md

```diff
@@ -124,6 +124,12 @@ Initialize Vault:
 vault operator init
 ```
 
+Unseal Vault (by supplying at least 3 keys from the above output)
+```
+vault operator unseal
+Unseal Key (will be hidden): <Enter key here>
+```
+
 Export the Vault token from the info from the initialize
 ```
 export VAULT_TOKEN=7a902c8b-a14e-f2af-e007-4482218c586e
```

Comment:
A very minor picky little thing...  The initial root token now looks like `s.KkNJYWF5g0pomcCLEmDdOVCW`.   (I was updating getting started guide, so happened to notice...)

Reply:
Good point, I can fix this now 👍